docs/ml-features.md

```diff
@@ -729,6 +729,58 @@ for more details on the API.
 </div>
 </div>
 
+## Interaction
+
+`Interaction` is a `Transformer` which takes a vector/double columns, and generate a single vector column that contains multiplication results of all combination of each vector/double values.
```

Comment:
Two small changes: "... which takes vector or double-valued columns, and generates a ..." 
"... contains the product of all combinations of one value from each input column"